api/spark.py

```
@@ -94,21 +94,28 @@ def run_spark(user_id: int, chat_id: int, conn, cfg) -> dict:
     idea_id = _store_spark_idea(conn, paper["id"], idea, idea_embedding, user_id)
 
+    try:
+        send_idea_message(
+            chat_id=chat_id,
+            idea_id=idea_id,
+            title=paper["title"],
+            url=paper["url"],
+            idea=idea,
+            bot_token=cfg.telegram_bot_token,
+        )
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)